.github/workflows/test.yml

```diff
@@ -0,0 +1,357 @@
+      needs.check-scope.outputs.test-handsontable-unit == 'true')
+    steps:
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # https://github.com/actions/checkout/releases/tag/v2.3.4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@25316bbc1f10ac9d8798711f44914b1cf3c4e954 # https://github.com/actions/setup-node/releases/tag/v2.4.0
+        with:
+          node-version: 15.9.0
+          cache: 'npm'
+      - run: npm ci --no-audit
+      - name: Download the Handsontable build artifact
+        uses: actions/download-artifact@3be87be14a055c47b01d3bd88f8fe02320a9bb60 # https://github.com/actions/download-artifact/releases/tag/v2.0.10
+        with:
+          name: handsontable-build-es-cjs
+          path: ./
+      - run: tar -zxf tmp.tar.gz ./tmp && rm tmp.tar.gz
+      - run: npm run swap-package-links
+      - run: npm run test:unit
+
+  test-handsontable-types:
+    name: "[TEST] Handsontable: types"
+    runs-on: ubuntu-latest
+    needs: [ check-scope, build-handsontable-es-cjs ]
+    if: |
+      needs.check-scope.outputs.run-all == 'true' ||
+      (needs.check-scope.outputs.skip-all == 'false' &&
+      needs.check-scope.outputs.test-handsontable-types == 'true')
+    steps:
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # https://github.com/actions/checkout/releases/tag/v2.3.4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@25316bbc1f10ac9d8798711f44914b1cf3c4e954 # https://github.com/actions/setup-node/releases/tag/v2.4.0
+        with:
+          node-version: 15.9.0
+          cache: 'npm'
+      - run: npm ci --no-audit
+      - name: Download the Handsontable build artifact
+        uses: actions/download-artifact@3be87be14a055c47b01d3bd88f8fe02320a9bb60 # https://github.com/actions/download-artifact/releases/tag/v2.0.10
+        with:
+          name: handsontable-build-es-cjs
+          path: ./
+      - run: tar -zxf tmp.tar.gz ./tmp && rm tmp.tar.gz
+      - run: npm run swap-package-links
+      - run: npm run test:types
+
+  test-handsontable-walkontable:
+    name: "[TEST] Walkontable"
+    runs-on: ubuntu-latest
+    needs: [ check-scope ]
+    if: |
+      needs.check-scope.outputs.run-all == 'true' ||
+      (needs.check-scope.outputs.skip-all == 'false' &&
+      needs.check-scope.outputs.test-handsontable-walkontable == 'true')
+    steps:
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # https://github.com/actions/checkout/releases/tag/v2.3.4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@25316bbc1f10ac9d8798711f44914b1cf3c4e954 # https://github.com/actions/setup-node/releases/tag/v2.4.0
+        with:
+          node-version: 15.9.0
+          cache: 'npm'
+      - run: npm ci --no-audit
+      - run: npm run test:walkontable
+
+  test-handsontable-e2e:
+    name: "[TEST] Handsontable: UMD"
```

Comment:
Should we call it E2E? It's using UMD build but it's testing e2e for all builds, right?

Reply:
I think it's using only the UMD build from `/dist/handsontable.js`.